doc/tls/README.md

```diff
@@ -0,0 +1,314 @@
+# TLS
+
+Skupper uses TLS certificates to authenticate and secure communications between
+Skupper routers in a network through channels called Links. The Skupper
+controller for Kubernetes will, by default, take care of issuing the self
+signed TLS credentials used by Links. This document describes the requirements
+for these TLS credentials, the default scheme used by the Skupper Kubernetes
+controller, and how other certificate infrastructure could be included.
+
+## TLS Credentials Requirements
+
+Skupper uses mutual TLS (mTLS) to establish Links between Sites. The Site that
+initiates the Link acts at the TLS client to the RouterAccess, which acts as
+the TLS server, The server side must present a valid server certificate and the
+client side must present its client certificate and both side validate each
+other's certificate (mTLS)  using a trusted CA``` CA. Any intermediate load
```

Comment:
> CA```` CA

typo?

Reply:
Oh boy! I've jumbled this paragraph accepting/merging review suggestions. Will take another look at this one!